client/dashboard/sites/index.tsx

```diff
@@ -42,7 +42,9 @@ const DEFAULT_FIELDS = [
 		label: __( 'URL' ),
 		enableGlobalSearch: true,
 		render: ( { item }: { item: Site } ) => (
-			<ExternalLink href={ item.URL }>{ new URL( item.URL ).hostname }</ExternalLink>
+			<ExternalLink href={ item.URL } style={ { overflowWrap: 'anywhere' } }>
```

Comment:
@youknowriad @ellatrix we noticed that we rarely are using (S)CSS files in the prototype and instead resort to this kind of inline style. Is this an intentional pattern?

Reply:
When it's layout, we often have it inline yes. It's similar to the stacking components I guess.

Regarding this rule, does it make sense to apply globally? There's URLs displayed in other places too.

Reply:
I don't mind small inline styles, but I have nothing against SCSS files. That said, I think custom CSS should kept to the minimum as much as possible. 

If something in the content of the different pages requires too much custom CSS, we should be raising flags about potential reusable components needs or alternatively rethink the design.

Reply:
As a suggestion, I’ve added the .break-anywhere class to the a element to prevent text overflow globally. Does it make sense? I'm unsure about the best practice for applying this type of rule globally 🤔

If it doesn't look good, I'd prefer to apply inline styles for now since only two places need this rule. We can reconsider this approach once we have more instances that require it.

Reply:
Maybe we add this style instead to `client/dashboard/app/style.scss`? 🤔 I'm not sure if it's a good idea to refer to a `@wordpress/components` class name here.

```css
a .components-external-link__contents {
    overflow-wrap: anywhere;
}

Reply:
^ Non-blocking; something that can be iterated later as needed.

Reply:
Thanks for fixing the other instance too. Not blocking, but I was actually wondering why we can't just apply `overflow-wrap: anywhere;` on the whole document. I'm curious what would break. :D

Reply:
Ideally, it would be better to apply `break-word` instead of `anywhere` on the whole documents. For example, if a link is `Learn more` , it would become something like 
```
Lear
n more.
```

It doesn't look good for users

Reply:
The problem with "reusing classes" lis that we end up with dead CSS code very quickly as there' no relation between the class definition and the component that actually use it.

All the following are good options to me:

- Inline style or a class that is specific to the current usage: `dashboard-sites__url-link` (a class that follows our CSS class naming guidelines and is tied to the component, so it's easy to remove when it becomes unecessary) 
- Absorbing this behavior in the `ExternalLink` component (not possible today because of the contribution pause)
- Writing an override of `ExternalLink` component maybe in `client/dashboard/external-link` with this

Reply:
> a class that is specific to the current usage: dashboard-sites__url-link

I considered this approach, but since multiple places would require the new class, I decided not to go with it.

> Absorbing this behavior in the ExternalLink component (not possible today because of the contribution pause)

😞

> Writing an override of ExternalLink component maybe in client/dashboard/external-link with this

That sounds like a good alternative. But since we only need it in two places for now, I'd prefer to use inline styles. We can revisit this approach if we end up needing it in more places 🙂